webpack.config.js

```diff
@@ -111,7 +111,6 @@ module.exports = (argvEnv, argv) => {
         let resolution = 'desktop'
         if (target.Chromium) buildTarget = 'chromium'
         if (target.Firefox) buildTarget = 'firefox'
-        if (target.FirefoxDesktop) firefoxVariant = 'fennec'
```

Comment:
I think `FirefoxDesktop` is not 'fennec', https://wiki.mozilla.org/Mobile/Fennec

Reply:
IGTM